README.md

```diff
@@ -27,6 +27,7 @@ These instructions will get you a copy of the project up and running on your loc
 * UNIX (Linux, Mac OSX), or Windows
 * [python](https://www.python.org/downloads/) >= 3.6
 * [SPECjbb®2015](https://www.spec.org/order.html)
+* [Tk](http://www.tkdocs.com/tutorial/install.html)
```

Comment:
Should mention this is only necessary for the gui component.

Reply:
This is for `dev-gui`, not `dev`. This won't go into `master`, but is a hard dependency for this branch, which is why I thought it would be good to add.

Reply:
Would an additional note for "if it's a GUI" be needed?